work-book

```diff
@@ -0,0 +1,8 @@
+Session 26/8
+	- created repo in github and maneged access 
+	- cloned the 6.824 mit repo
+	- installed golang 
+	- built and run the sample project of lab 01 map reduce
+	- agreed to read the https://pdos.csail.mit.edu/6.824/labs/guidance.html
+	- agreed to start the go course https://learning.oreilly.com/library/view/learning-go/9781492077206/
```

Comment:
Url should be replaced with https://learning.oreilly.com/videos/ultimate-go-programming/9780135261651/

Reply:
Done